air/src/air/trace_info.rs

```diff
@@ -198,9 +201,8 @@ impl TraceInfo {
         }
     }
 
-    /// Returns the number of columns in the auxiliary trace segment.
-    pub fn get_aux_segment_width(&self) -> usize {
-        self.aux_segment_width
+    pub fn is_with_logup_gkr(&self) -> bool {
+        self.logup_gkr
     }
```

Comment:
Let's add a doc comment to this method.

Also, maybe something like `logup_gkr_enabled()` could be a better method name? Not a strong preference though - so, feel free to ignore.

Reply:
Changed